cypress/e2e/functional/researcher_permission_forms.spec.js

```diff
@@ -0,0 +1,113 @@
+
+        cy.log('Fill in the Create New Permission Form dialog again, but this time, click the Save button');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+
+        cy.log('Create more Permission Forms, to be used to test the Archive, Unarchive & Delete buttons');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'4', 'learn.portal.staging.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+            
+        cy.log("Select a Project from the dropdown (on the 'Create New Permission Form' tab)");
+        pf.selectCreateManagePermissionsProject(TEST_PROJECT_NAME);
+
+        cy.log('Test the links on the Permission Forms table, starting with URL text, Archive, and Unarchive');
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
+
+        // TODO: fix clicking on archived rows, then uncomment these:
+        // pf.clickPermissionFormArchive(TEST_PERMISSION_FORM_NAME+'2');
+        // pf.clickPermissionFormArchive(TEST_PERMISSION_FORM_NAME+'3');
+        // pf.clickPermissionFormUnarchive(TEST_PERMISSION_FORM_NAME+'3');
+
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL);
+
+        cy.log('Test the Edit link, for an Archived, an Unarchived and a never-Archived Permision Form');
+        pf.clickPermissionFormEdit(TEST_PERMISSION_FORM_NAME+'2');
+        pf.verifyEditPermissionFormDialog(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL);
+        pf.clickCreateEditPermissionFormCancelButton();
+
+        pf.clickPermissionFormEdit(TEST_PERMISSION_FORM_NAME+'3');
+        pf.verifyEditPermissionFormDialog(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
+        pf.clickCreateEditPermissionFormCancelButton();
+
+        pf.clickPermissionFormEdit(TEST_PERMISSION_FORM_NAME+'1');
+        pf.verifyEditPermissionFormDialog(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL);
+        pf.clickCreateEditPermissionFormCancelButton();
```

Comment:
@johnTcrawford we are only covering the scenario to test whether the edit dialog is displayed with all the correct information but there is no coverage to ensure that editing the permission form and saving is saved those changes.

Reply:
Yes, see the TODO immediately below. This is also one of the tasks of PT #187863588.